R/quiz.R

```diff
@@ -304,7 +304,7 @@ knit_print.tutorial_question <- function(x, ...) {
   rmarkdown::shiny_prerendered_chunk(
     'server',
     sprintf(
-      'learnr:::question_prerendered_chunk(%s)',
+      'learnr:::question_prerendered_chunk(%s, session)',
```

Comment:
Matches the named arg after a `...`

```suggestion
      'learnr:::question_prerendered_chunk(%s, session = session)',
```

Reply:
Good catch, thank you!